File: Hybrid_Task/Python/Dinner/lcm.py
number_one = int(input("Enter a number: "))
number_two = int(input("Enter another number: "))

multiple = 1

while multiple != 0:
	if multiple % number_one != 0 or multiple % number_two != 0:
		multiple += 1
	if multiple % number_one == 0 and multiple % number_two == 0:
		break

print(multiple)

File: Hybrid_Task/Java/Dinner/LCM.java
import java.util.Scanner;

public class LCM {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.print("Enter a number: ");
int numberOne = input.nextInt();

System.out.print("Enter another number: ");
int numberTwo = input.nextInt();

int multiple = 1;
while(multiple != 0) {
	if(multiple % numberOne != 0 | multiple % numberTwo != 0) multiple++;
	if(multiple % numberOne == 0 & multiple % numberTwo == 0) break; 
}
System.out.print(multiple);
}
}